README.md

```diff
@@ -1,35 +1,17 @@
-HTTP Endpoints
-==============
+# Introduction
 
-Base URL: `http://hyper-recipes.herokuapp.com`
+This is the backend for the Hyper Recipes app, to be able to use it you first have to register and use the provided token, this is helpful so other applicants don't modify your recipes in any way.
 
-## Create user
+The base HTTP endpoints is: `http://hyper-recipes.herokuapp.com`
+
+## Register to get a token
 
 * **email:string** (obligatory field, must be valid email format)
 * **password** (obligatory field)
 * **password_confirmation** (obligatory field)
 * seed_recipes:boolean
 
-### without dummy recipes
```

Comment:
I removed this in order to make things less confusing, also I don't see why people would want to make recipes without dummy data.